src/code/z_play.c

```diff
@@ -6,7 +6,7 @@ void* D_8012D1F0 = NULL;
 UNK_TYPE D_8012D1F4 = 0; // unused
 Input* D_8012D1F8 = NULL;
 
-TransitionStruct D_801613B0;
+FBFilter D_801613B0;
```

Comment:
Do you have any idea of a name for this `D_801613B0` variable? It would be nice to name it with these changes, if possible, even if it's something basic or default.

Reply:
I’d say something like trnsnDefault and the control trnsnDefaultState